### ISSUE TITLE
artifactPath being changed in plugin, can break build

### ISSUE THREAD

**Issue by aappddeevv** [swachter/scala-ts#1]
```scala
artifactPath in fastOptJS in Compile := scalaTsOutputDir.value / (scalaTsFilenamePrefix.value + ".js"),
artifactPath in fullOptJS in Compile := scalaTsOutputDir.value / (scalaTsFilenamePrefix.value + ".js"),
```
Is there a way to do this without changing the artifactPath value? That's changed by other plugins I have as part of a unified build.

**Comment by swachter** [comment]
Good point. I think that the ScalaTsPlugin does not need output configuration.

It can write the .d.ts file in the directory the JavaScript output is written to. In addition, the generated package.json can always reference the JavaScript artifact file name.

In addition, I think it would be best, if the scalaTsPackage task would be automatically added after the fullOptJS and fastOptJS tasks.

**Comment by swachter** [comment]
The ScalaTsPlugin does no more configure the artifactPath. It uses the artifactPath of ScalaJS and places its files as siblings.

The crossTarget setting can be used to configure the output directory of fastOptJS/fullOptJS thereby also configuring the ScalaTs output.

(Released in 0.3)
